### PR TITLE
Add a larger paid plan

### PR DESCRIPTION
**Pull request by ianopolous**
The two plan sizes are displayed next to each other, or wrap on smaller screens. 

When upgrading or updating a card, the plans disappear to make sure the card iframe is visible.
Screenshot 2022-03-17 at 22-45-15 Peergos
Screenshot 2022-03-17 at 22-45-28 Peergos
Screenshot 2022-03-17 at 22-46-17 Peergos

### CODE REVIEW

File: src/components/modal/ModalPro.vue [hunk 1]
```diff
@@ -7,27 +7,36 @@
...
-
-			</div>
+                                <AppButton @click.native="updateCard(53687091200)" :disabled="isPro" type="primary" block accent>{{proButtonText}}</AppButton>
+			    </div>
+                            <div class="card__meta options">
+				<h3>Visionary Account</h3>
+				<ul>
+				    <li>500 GB of hyper secure storage</li>
+				    <li>All our bundled private applications</li>
+				    <li>25£ / month</li>
```
Comment (kevodwyer) [inline src/components/modal/ModalPro.vue:26]: use &#x00A3; for uk pound symbol

File: src/components/modal/ModalPro.vue [hunk 2]
```diff
@@ -117,7 +140,7 @@ module.exports = {
 			if (quotaBytes >= bytes && bytes > 0) {
 			    that.updatePayment()
 			    that.$store.commit("SET_MODAL", false)
-			    that.$toast.info('Thank you for signing up to a Peergos Pro account!',{timeout:false, id: 'pro'})
```
Comment (kevodwyer) [inline src/components/modal/ModalPro.vue:120]: maybe add an exceptionally block on requestSpace to handle unforseen/network errors. Nothing worse then not knowing if request completed.

File: src/components/modal/ModalPro.vue (hunk 1 above)
Comment (kevodwyer) [reply src/components/modal/ModalPro.vue:26]: i mean the numeric character reference
\# &#x00A3;

Comment (ianopolous) [reply src/components/modal/ModalPro.vue]: done

**Review by kevodwyer — APPROVED** [review]

**Pull request merged**